app/src/lib/git/status.ts

```diff
@@ -202,18 +202,25 @@ export async function getStatus(
   }
 
   const mergeHeadFound = await isMergeHeadSet(repository)
+  const conflictedFilesInIndex = entries.some(e => e.statusCode === 'UU')
```

Comment:
should we also look for the other conflicted codes listed in [status-parser](https://github.com/desktop/desktop/blob/189d17f44a4744db26e1ce69b658b9cc98f3aba1/app/src/lib/status-parser.ts#L301-L362)? i'm not totally sure if they can all be generated by a stash pop but i'd rather be safe.

* `DD`
* `AU`
* `UD`
* `UA`
* `DU`
* `AA`
* `UU`

Reply:
Good shout! I'll update this tomorrow to try and figure out the most optimal way to do a broad check like this.